tests/ui_tests/test_service_page.py

```diff
@@ -248,6 +265,96 @@ def test_save_custom_config_on_service_config_page(self, app_fs, sdk_client_fs):
         service_config_page.config.save_config()
         service_config_page.config.compare_versions(params["config_name_old"])
         service_config_page.config.check_config_fields_history_with_test_values()
+        with allure.step("Check invisible params"):
+            config = service.config()
+            assert len(config.keys()) == 28, "There are should be 24 config parameters"
```

Comment:
please make a constant: in one place it's 28, in another it's 24

Reply:
added const